src/fonduer/learning/disc_models/logistic_regression.py

```diff
@@ -4,23 +4,37 @@
 from scipy.sparse import issparse
```

Comment:
`get_train_idxs` takes `rand_state`, but it is not described in the docstring. You can add types to the parameters too.

Other than that LGTM.

Reply:
Done!